classes/class-loader.php

```diff
@@ -110,6 +110,18 @@ private function __construct() {
 
 		// Load block patterns.
 		Block_Pattern::register_all();
+
+		add_action(
```

Comment:
This of course will only be needed if we end up using the Rellax library ([see PR](https://github.com/greenpeace/planet4-master-theme/pull/1783)). I'm not sure if there's a better place to add this though 🤔

Reply:
The other conditional enqueues we have are each in their own block class (carouselheader, gallery), so I guess this one is fine here :) We'll probably rework/regroup those if we try to systematically split and load css/js conditionally.
You can also use `BlockList::has_block()` instead of `has_block()` if you want to handle the reusable block issue.